compiler/src/dotty/tools/dotc/core/StdNames.scala

```diff
@@ -436,6 +435,7 @@ object StdNames {
     val implicitConversions: N  = "implicitConversions"
     val implicitly: N           = "implicitly"
     val in: N                   = "in"
+    val `inline`: N             = "inline"
```

Comment:
Is this necessary with this change? Is it a style we want to recommend for soft keyword?

Reply:
It's no longer necessary. I removed the backticks.